ReactAndroid/src/main/java/com/facebook/react/modules/storage/AsyncStorageModule.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
why touch this line.

Reply:
Android Studio was showing warning, so fixed it 😄